addons/web/static/tests/webclient/actions/embedded_action.test.js

```diff
@@ -357,3 +357,46 @@ test("the embedded actions should not be displayed when switching view", async (
         message: "The embedded actions menu should not be displayed",
     });
 });
+
+test("main_embedded_action_movable context key should allow the user to move the main embedded action", async () => {
+    await mountWithCleanup(WebClient);
+    await getService("action").doAction(1, {
+        additionalContext: {
+            main_embedded_action_movable: true,
+        },
+    });
+    browser.localStorage.clear();
+    await contains(".o_control_panel_navigation > button > i.fa-sliders").click();
+    await contains(".o_embedded_actions .dropdown").click();
+    await contains(
+        ".o_popover.dropdown-menu .dropdown-item > div > span:contains('Embedded Action 2')"
+    ).click();
+    browser.ontouchstart = true; // hack to make "waitForTouchDelay" effective
```

Comment:
Have you tried with `mockTouch()` ?

Reply:
Looks good with `mockTouch()`, thank you